tests/pytests/test_issues.py

```diff
@@ -801,6 +801,21 @@ def test_mod5252(env):
   env.assertEqual(res, [1, ['key_name', 'doc']])
 
 
+@skip(cluster=True)
+def test_mod_6276(env):
+  # Setting the gc threshold to 0 so the gc won't skip its periodic run
+  env.expect('FT.CONFIG', 'SET', 'FORK_GC_CLEAN_THRESHOLD', '0').ok()
```

Comment:
Maybe we should also configure the GC interval to lower value so the test will finish faster?

Reply:
We don’t need to. That’s why we stop it in the first place. We trigger it immediately when we need to